todoman/model.py

```diff
@@ -237,7 +237,7 @@ class Database:
     def __init__(self, path):
         self.path = path
 
-    @cached_property
+    @property
```

Comment:
I think this was done to keep the database fresh even after editing an item. Let's make `todos` not a property, but instead rename it to `get_todos` with a `skip_cache` parameter.